examples/step-37/doc/intro.dox

```diff
@@ -473,3 +484,44 @@ settings (on the command line, specify
 <tt>-DCMAKE_CXX_FLAGS="-march=native"</tt>, see the deal.II README for more
 information). Similar options exist for other compilers.
 
+
+<h3>Running multigrid on large-scale parallel computers</h3>
+
+As mentioned above, all components in the matrix-free framework can easily be
+parallelized with MPI using domain decomposition. Thanks to the good support
+of large-scale parallel mesh framework provided by p4est (see step-40 for
+details) in deal.II, and the fact that cell-based loops with matrix-free
+evaluation <i>only</i> need a decomposition of the mesh into chunks of roughly
+equal size on each processor, there is relatively little to do for extending
```

Comment:
maybe don't say `extending` as you don't extend any serial matrix-free tutorial here. Maybe something like
`...little to do to write distributed memory parallel programs`?